### PR TITLE
Document Update

### PR DESCRIPTION
**Pull request by NiteshKoushik**
Keyword change from "scalably"  to "scalable"

### CODE REVIEW

File: docs/modules/ROOT/pages/basic-concepts.adoc [hunk 1]
```diff
@@ -3,7 +3,7 @@
...
 
-As reported on the Apache Kafka website: _Kafka Connect is a tool for scalably and reliably streaming data between Apache Kafka and other systems. It makes it simple to quickly define connectors that move large collections of data into and out of Kafka. Kafka Connect can ingest entire databases or collect metrics from all your application servers into Kafka topics, making the data available for stream processing with low latency. An export job can deliver data from Kafka topics into secondary storage and query systems or into batch systems for offline analysis._
+As reported on the Apache Kafka website: _Kafka Connect is a tool for scalable and reliably streaming data between Apache Kafka and other systems. It makes it simple to quickly define connectors that move large collections of data into and out of Kafka. Kafka Connect can ingest entire databases or collect metrics from all your application servers into Kafka topics, making the data available for stream processing with low latency. An export job can deliver data from Kafka topics into secondary storage and query systems or into batch systems for offline analysis._
```
Comment (orpiske) [inline docs/modules/ROOT/pages/basic-concepts.adoc:6]: In this case, I think it should be rephrased as "scalable and reliably data streaming between" ...

**Review by orpiske — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/basic-concepts.adoc (hunk 1 above)
Comment (oscerd) [reply docs/modules/ROOT/pages/basic-concepts.adoc:6]: Yes

**Review by oscerd — CHANGES_REQUESTED** [review]
+1 for otavio suggestion

**Comment by NiteshKoushik** [comment]
Changes made as per the comments

**Review by orpiske — APPROVED** [review]
Thanks for the changes. LGTM.

**Review by oscerd — APPROVED** [review]

**Pull request merged**